src/point.class.ts

```diff
@@ -351,6 +352,14 @@ export class Point {
     return new Point(this.x, this.y);
   }
 
+  /**
+   * return the magnitude of a vector
+   * @return {number}
+   */
+  magnitude(): number {
```

Comment:
is it the distance? can we add in the JSDOC ` return the magnitued ( or distance from origin ) of a point` i m not sure we should refer a vector and a point like if they are the same thing

Reply:
> i m not sure we should refer a vector and a point like if they are the same thing

I agree.
Maybe I refactor Point and extract Vector in a follow up

Reply:
no just call it a point, and jsut write somewhere that this represent the point distance from the origin, OR the magnitude of a vector that has origin on 0,0 and direction expressed by point p.
We don't need to refactor, just a different wording.

Reply:
What about moving it as a util yo vectors?
I prefer that